docs/contributing.rst

```diff
@@ -69,25 +69,25 @@ In order to have a copy of the repo on your local computer and be able to render
   :align: center
   :alt: decorative line
 
-2. After you successfully fork the repo, you want to **clone the repo** on your local computer. From there, you can make edits using your favorite source code text editor, such as `Sublime Text <https://www.sublimetext.com/>`_.
+2. After you successfully fork the repo, you want to **clone your fork of the repo** on your local computer. From there, you can make edits using your favorite text editor.
 
-* Go to where you just forked the handbook repo on your Github repository.
-* Click on "Clone or download" and copy the URL to clone the repo.
+* Navigate to the forked repo on your GitHub account.
+* Click on "Clone or download" and copy the URL.
 
 .. figure:: images/clone_repo.png
   :width: 700px
   :align: center
   :alt: clone the repo!
 
   
-*The image above, the red box highlights the button you need to press to get the URL to clone the repo. The arrow points to info about the location of the original repo you forked.*
+*In the image above, the red box highlights the button you need to press to get the URL to clone the repo. The arrow points to info about the location of the original repo you forked.*
 
 * In your terminal window, change directory to the location where you want the clone of the repo to live. Then, clone the repo using the URL you copied in the previous step.
 
 .. code-block:: bash
 
     # change directory to the place where you want the cloned repo
-    cd Desktop/princeton_handbook/
+    cd ~/Desktop/princeton_handbook/
```

Comment:
We should specify "(on Mac)" somewhere here... Also, do we really want to create this `~/Desktop/princeton_handbook/handbook/`... nested structure? For example something like `~/Desktop/handbook` would be sufficient, or (my example) `~/Work/brainhack-princeton/handbook` or just `~/Work/handbook`

Reply:
I added "(e.g., on Mac)"... If nobody has strong feelings about the directory structure, we can ignore this comment since it doesn't really matter!